packages/combobox/style.module.css

```diff
@@ -19,6 +19,8 @@
       box-shadow: var(--select-shadow);
       background: var(--white);
       border-radius: 2px;
+      max-height: var(--combobox-list-max-height, 40vh);
+      overflow: scroll;
```

Comment:
@jmfury Can we use `overflow-y: auto` here? That way the scroll bar never shows horizontally and only vertically when needed.

Reply:
@alexcarpenter Great call - silly oversight on my part. Can you to throw up a PR for a patch? -- if not I can, but won't get to it til Monday -- moving onto some other stuff atm!